### PR TITLE
[validator] Add second argument `options` for isCreditCard

### PR DESCRIPTION
**Pull request by babiakrs**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [validator.js docs](https://github.com/validatorjs/validator.js/blob/master/README.md), `isCreditCard` method

### CODE REVIEW

File: types/validator/index.d.ts [hunk 1]
```diff
@@ -271,10 +272,17 @@ declare namespace validator {
      */
     function isByteLength(str: string, options?: IsByteLengthOptions): boolean;
 
+    interface IsCreditCardOptions {
+        /**
+         * @default undefined
+         */
+        provider?: 'amex' | 'dinersclub' | 'discover' | 'jcb' | 'mastercard' | 'unionpay' | 'visa';
```
Comment (peterblazejewicz) [inline types/validator/index.d.ts:279]: docs mentions:

> or blank 

can you check this with source what does it mean? e.g. '' is a blank, thanks!

Comment (Almost-Infinity) [reply types/validator/index.d.ts]: It means any falsy value. I've added an empty string as a type for the 'blank' value.

**Comment by peterblazejewicz** [comment]
LGTM!
@Almost-Infinity thanks!

**Review by peterblazejewicz — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge

**Pull request merged**